packages/edge-gateway/src/gateway.js

```diff
@@ -537,7 +537,7 @@ function getTransformedResponseWithCustomHeaders(response) {
 
   clonedResponse.headers.set(
     'content-security-policy',
-    "default-src 'self' 'unsafe-inline' blob: data: ; form-action 'self' ; navigate-to 'self' "
+    "default-src 'self' 'unsafe-inline' 'unsafe-eval' blob: data: https://*.githubusercontent.com; form-action 'self' ; navigate-to 'self'; connect-src https://polygon-rpc.com "
```

Comment:
```suggestion
    "default-src 'self' 'unsafe-inline' 'unsafe-eval' blob: data: https://*.githubusercontent.com; form-action 'self' ; navigate-to 'self'; connect-src https://polygon-rpc.com https://rpc.testnet.fantom.network "
```

Looks like we also need fantom per https://github.com/nftstorage/nft.storage/issues/2143#issuecomment-1226617976